financialaid/migrations/0005_modified_financial_aid_audit.py

```diff
@@ -9,7 +9,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('financialaid', '0003_added_timestamps'),
+        ('financialaid', '0004_currencyexchangerate'),
```

Comment:
Please put the new migration after all other existing ones, and leave the old migrations untouched